src/content/app-platform/apps/kong/index.md

```diff
@@ -0,0 +1,46 @@
+---
+linkTitle: Kong
+title: Kong API Gateway
+description: An overview of how and what Giant Swarm offers with the Kong API Gateway.
+weight: 60
+menu:
+  main:
+    parent: app-platform-apps
+user_questions:
+  - What is tested for Kong?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-cabbage
+last_review_date: 2022-02-23
+---
+
+# Kong
```

Comment:
You should remove the headline, as it will appear as part of the template, using `title` from the front matter.